scripts/jq/1.5-239278fd/script.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+MASON_NAME=jq
+MASON_VERSION=239278fd3a02dc1ae0521a7a57d573dbf977b2d9
+MASON_LIB_FILE=bin/jq
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/${MASON_NAME}-${MASON_VERSION}
+
+    git clone git://github.com/stedolan/${MASON_NAME}.git ${MASON_BUILD_PATH}
+    cd ${MASON_BUILD_PATH}
+    git checkout ${MASON_VERSION}
+
+    cd ../
+}
+
+function mason_compile {
+    git submodule update --init
+    autoreconf -fi
+
+    ./configure \
+        --prefix ${MASON_PREFIX} \
+        --with-oniguruma=builtin \
+        --disable-maintainer-mode
+
+    make -j${MASON_CONCURRENCY} LDFLAGS=-all-static
```

Comment:
Please do `LDFLAGS="<your custom flag>" ${LDFLAGS:-}` instead. This is important since Mason on some platforms may be customizing LDFLAGS and we want to respect the mason provided environment values as well.